docs/en/Contributing-Getting-Started.md

```diff
@@ -267,6 +271,10 @@ into the Alluxio code base.
 
 Congratulations! You have successfully contributed to Alluxio! Thank you for joining the community!
 
+## <a name="video"></a>Video
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/QsbM804rc6Y" frameborder="0" allowfullscreen></iframe>
```

Comment:
If we don't have anything out of the box, let's treat this as a one-off. If we see the need for more embedded videos in our docs we can consider adding something to support the syntax.